manifests/b/Borvid/HttpMasterExpress/5.5.0/Borvid.HttpMasterExpress.locale.en-US.yaml

```diff
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=AUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Borvid.HttpMasterExpress
+PackageVersion: 5.5.0
+PackageLocale: en-US
+Publisher: Borvid
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+Author: Borvid
+PackageName: HttpMaster Express Edition
+PackageUrl: https://www.httpmaster.net
+License: Copyright (C) 2022 Borvid
```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate